index.bs

```diff
@@ -456,37 +468,30 @@ origin, as described in [[#serializing]].
 
 Issue: TODO(jww): Formal definition of the headers and responses w/grammars.
 Also need to be explicit about `*` having same limitations as
-`Access-Control-Allow-Origin` w/credentials. Also need to describe opt-outs to
-use old-school Origin header and ACAO header.
+`Access-Control-Allow-Origin` w/credentials. 
```

Comment:
I was thinking that should go into the whole opt-outs section, rather than here (definition of headers etc)?